### PR TITLE
Update Russian version of collections.md

### PR DESCRIPTION
**Pull request by gaynetdinov**
Some suggestions and improvements as well as typo fixes. 

It's really hard to come up with Russian translation for 'keyword list' term.

### CODE REVIEW

File: ru/lessons/basics/collections.md [hunk 1]
```diff
@@ -95,9 +95,9 @@ iex> File.read("path/to/unknown/file")
 {:error, :enoent}
 ```
 
-## Списки с ключами
+## Ключевые списки
```
Comment (brain-geek) [inline ru/lessons/basics/collections.md]: https://github.com/doomspork/elixir-school/blob/master/GLOSSARY.md#elixir-related-terms

> keyword list - список с ключами

That's current translation in glossary. Please change it there too.

**Comment by brain-geek** [comment]
Wow. Thanks, @gaynetdinov .

**Comment by gaynetdinov** [comment]
@brain-geek didn't know about glossary, sorry. I fixed translations for `keyword list`.

**Comment by brain-geek** [comment]
LGTM.

**Comment by doomspork** [comment]
:boom:

Awesome @gaynetdinov and @brain-geek.  Thank you so much.  Apologizes for the delays, I'm currently in the middle of traveling.

**Pull request merged**